scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -127,27 +104,23 @@ def test_support_alternative_backends(xp, f_name, n_args, dtype, shapes):
         # Try to trigger bugs related to having multiple chunks.
         args_xp = [arg.rechunk(5) for arg in args_xp]
 
-    res = fw(*args_xp)
-    ref = f(*args_np)
+    res = nfo.wrapper(*args_xp)  # Also wrapped by lazy_xp_function
```

Comment:
To test `lazy_xp_function`:
1. go to `xlogy`, add the line `assert xp.any(x) or ~xp.any(x)`, and watch Dask fail with spurious compute error
2. go to `stdtr`, add the line `assert xp.any(t) or ~xp.any(t)`, and watch JAX fail with ConcretizationError